### ISSUE TITLE
.first() and .last() throw `TypeError: 'undefined' is not an object`

### ISSUE THREAD

**Issue by thom-nic** [jquense/teaspoon#11]
when `.find(selector)` returns a collection of length 0, calling `.first(selector)` or `.last(selector)` throw

```
	TypeError: 'undefined' is not an object (evaluating 'inst._isQueryCollection')
	    at isQueryCollection (:29)
	    at $init (:49)
	    at anonymous (:1)
	    at :145
	    at :155
	    at QueryCollection (:21)
	    at first (:93)
	    at first (:93)
	    at :46
```

(sorry for the bad backtrace line numbers.)  I would expect them to return `null` or `undefined` or throw a new error message like `"No element found matching selector 'selector'"`

**Comment by jquense** [comment]
fixed! sorry forgot to ref in the commit. thanks

**Comment by thom-nic** [comment]
Yeah this seemed to disappear after you fixed #13.  Thanks again!
